src/translate_source_keys.py

```diff
@@ -32,7 +33,7 @@ def set_show_ids(cls, user, data, pipeline_configuration):
                 if not remapping.is_activation_message:
                     continue
 
-                if td.get(remapping.source_key) is not None:
+                if remapping.source_key in td:
```

Comment:
It might seem surprising that this needs to change. It needs to change because the old version didn't handle runs where the text field was null. I've tested by re-running the pipeline over all of the data and confirming the results were the same, before introducing the timestamp remapping.

Reply:
lgtm